docs/src/modules/components/Link.js

```diff
@@ -71,9 +73,11 @@ Link.defaultProps = {
   activeClassName: 'active',
 };
 
-export default compose(
+const RouterLink = compose(
   withRouter,
   connect(state => ({
     userLanguage: state.options.userLanguage,
   })),
 )(Link);
+
+export default React.forwardRef((props, ref) => <RouterLink {...props} innerRef={ref} />);
```

Comment:
withRouter from next/router does not support ref forwarding nor exposes a helper prop.